charts/catalog/templates/rbac.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+kind: List
+items:
+
+# TODO: if this is just for namespace lifecycle admission, move to a generic role
+- apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
I believe this should be the only use for the API server, and does constitute what is necessary for namespace lifecycle admission.  I'm cool calling this role something like `namespace-lifecycle-admission`, your choice.